apps/wallet-mock/src/test/wallets.test.ts

```diff
@@ -0,0 +1,84 @@
+import { test, expect } from '@playwright/test'
+import { mock } from '../support/mock'
+import adminMock from '../types/admin-mock'
+
+test.describe('Wallets view', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/')
+  })
+
+  test('should see wallets header', async ({ page }) => {
+    await expect(
+      page.getByTestId('wallet-home').getByRole('heading')
+    ).toHaveText('Wallets')
+  })
+
+  test('should see wallets list', async ({ page }) => {
+    await expect(page.getByTestId('wallet-list')).toBeVisible()
```

Comment:
are there any plans to declare these kind of locators in one place only? Ideal if so because from experience multiple declarations of things like this can lead to a bit of a maintenance nightmare. I think there should be a single source of truth for locators and functionality..